src/styles/_Card.scss

```diff
@@ -1,22 +1,5 @@
 $card-text-color: #FFFF00;
 
-@keyframes bounce {
-  0%, 20%, 60%, 100% {
-    -webkit-transform: translateY(0);
-    transform: translateY(0);
-  }
-
-  40% {
-    -webkit-transform: translateY(-10px);
-    transform: translateY(-10px);
-  }
-
-  80% {
-    -webkit-transform: translateY(-10px);
-    transform: translateY(-10px);
-  }
-}
-
```

Comment:
i may have merged this back into the repo when i did my last commit. we may have to take it back out if we want it out.